osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/services/RedisPermitService.java

```diff
@@ -65,22 +70,40 @@ public boolean createPermit(
 
           log.debug(
               "Trying to register a permit for request[{}] with permit key {}. (max-concurrent-requests: {}, number-of-registered-permits: {})",
-              requestId,
+              permitRequest.getRequestId(),
               permitKey.key(),
               maxConcurrentRequests,
               numberOfRegisteredPermits);
 
-          if (maxConcurrentRequests < 0 || numberOfRegisteredPermits < maxConcurrentRequests) {
-            granted =
-                permits.add(
-                    Instant.now().toEpochMilli(), new Permit(networkSegment, clientId, requestId));
+          // get waiting room for key
+          final RScoredSortedSet<PermitRequest> lobby =
+              this.redisson.getScoredSortedSet(permitKey.lobby());
+
+          // if prio = default and waiting room size > 0 => EXIT (granted = false)
```

Comment:
If this  comment is explicitely implemented you don't need the comment.

Reply:
removed comments